src/main/java/com/google/sps/utils/GameUtils.java

```diff
@@ -0,0 +1,43 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import com.google.appengine.api.datastore.Entity;
+import java.util.ArrayList;
+
+public final class GameUtils {
+
+  /**
+  * When a user creates or joins a game, checks to see whether the user already has a game by that name, and returns false if so
```

Comment:
"When a user creates or joins a game"  -- please leave this phrase out. The goal is to document IsvalidGameName, not the code that will call IsValidGameName. Otherwise you'll confuse the reader.

Reply:
Got it, that's done. Thanks!